tensorflow/lite/delegates/openvino/operations/add.cc

```diff
@@ -0,0 +1,25 @@
+#include "add.h"
+
+namespace tflite {
+namespace openvinodelegate {
+
+std::shared_ptr<ov::Node> Add::createNode() {
+  TfLiteAddParams* add_params = (TfLiteAddParams*)GetBuiltinData();
+  auto inputNode1 =
+      getInputNode(tensor_indices[0]);
+  if (inputNode1 == nullptr)
+      TFLITE_LOG(INFO) << "input node 1 is null\n";
```

Comment:
We need a way to pass the error status when creating the node. This logging is not useful. Should we return TfLiteStatus and maintain the state for the added Node to access via getAddedNode.